xml/ay_bigfile.xml

```diff
@@ -9996,9 +9996,9 @@ config:type="boolean"&gt;true&lt;/keep_install_network&gt;</screen>
 
      <para>
       It is possible to specify <literal>rootpassword</literal> in
-      linuxrc and also have a user section for the root user. If this
```

Comment:
we also have an entity for the user 'root' that tags the word 'root' as systemitem -> "for the &rootuser; user"
(unfortunately, it is a bit clumsy to use in the text as the entity as such is called 'rootuser' but only resolves to 'root' which makes it awkward in the source text because you have to add the word 'user' afterwards)

Reply:
That is pre-existing text -- I probably just edited it for indentation. But a good catch all the same. Fixed.